### DISCUSSION TITLE
Keyboard can't input for from winx32 to some host

### DISCUSSION THREAD

**Discussion by lgzcoollg** [rustdesk/rustdesk#9612] [Q&A] [answered]
**ENV:**
Host: 
win7 x32, rustdesk 1.2.8/1.2.9/1.3.0

Remote:
win10 x64, rustdesk 1.2.8/1.2.9/1.3.0
macOS, rustdesk 1.2.9/1.3.0

1. I try from android and macOS host to control at the same time, it can input!
2. I try to change the other input modes(lacy, 1:1), still can't input.
3. I try to upgrade the version of rustdesk both side, still can't input.
4. I try Clipboard, It can use between 2 system!
5. I try to control another winx64(rustdesk 1.2.3), it can input!

**Comment by fufesou** [comment, marked as answer]
Please try `1.3.1` or the latest nightly build

https://github.com/rustdesk/rustdesk/discussions/9143

**Reply by lgzcoollg** [reply]
it fixed at ### 1.3.1, thanks!
